test/studies/isx/PERFEXECOPTS

```diff
@@ -1 +1 @@
---mode=weakISO --useSubTimers=true
+--mode=weakISO --useSubTimers=true --n 8388608
```

Comment:
Offline we agreed to set perBucketMultiply to 1 for both single and multi-node testing.